docs/pages/reference/backends.mdx

```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
 
 ## Auth Server State
 
-To run multiple instances of Teleport Auth Server, you must switch to a High Availability secrets back-end first.
+To run multiple instances of Teleport Auth Server, you must switch to a High
+Availability secrets back-end first.
 
-To do this, use a load balancer to create a single auth API access point (AP) and specify this AP in the `auth_server` field of the Teleport configuration file for all Nodes in a cluster. This load balancer should do TCP-level forwarding.
+To do this, use a load balancer to create a single auth API access point (AP)
```

Comment:
Does "auth API access point" mean the same thing as "a single access point for the Auth Service API"?